### PR TITLE
translate  content/zh-cn/blog/_posts/2022-08-25-pod-security/ into Chinese

### PR DESCRIPTION
**Pull request by sarazqy**
translate  content/zh-cn/blog/_posts/2022-08-25-pod-security/ into Chinese
Fixes : https://github.com/kubernetes/website/issues/36442

### CODE REVIEW

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+removed.
+-->
+Kubernetes v1.25 的发布标志着 Kubernetes 开箱即用的 Pod 安全控制的一个重要里程碑：Pod 安全准入（PSA）逐渐稳定，
+而 Pod 安全策略（PSP）已被移除。
+
+<!--
+[PSP was deprecated in Kubernetes v1.21](https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/),
+and no longer functions in Kubernetes v1.25 and later.
+-->
+[PSP 在 Kubernetes v1.21 中被弃用](https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/)，之后在 Kubernetes v1.25 中不再起作用。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:32]:
```suggestion
[PSP 在 Kubernetes v1.21 中被弃用](/zh-cn/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/)，在 Kubernetes v1.25 及更高版本中不再起作用。
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+<!--
+The Pod Security admission controller replaces PodSecurityPolicy, making it easier to enforce predefined
+[Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) by
+simply adding a label to a namespace. The Pod Security Standards are maintained by the K8s
+community, which means you automatically get updated security policies whenever new
+security-impacting Kubernetes features are introduced.
+-->
+Pod 安全准入控制器取代了 PodSecurityPolicy，只需向命名空间添加标签，就可以更轻松地执行预定义的 [Pod 安全标准](https://kubernetes.io/docs/concepts/security/pod-security-standards/)。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:41]:
```suggestion
Pod 安全准入控制器取代了 PodSecurityPolicy，只需向命名空间添加标签，
就可以更轻松地执行预定义的 [Pod 安全标准](/zh-cn/docs/concepts/security/pod-security-standards/)。 
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+<!--
+## What’s new since Beta?
+
+Pod Security Admission hasn’t changed much since the Beta in Kubernetes v1.23. The focus has been on
+improving the user experience, while continuing to maintain a high quality bar.
+-->
+## 自 Beta 版以来有什么新功能？
+
+自 Kubernetes v1.23 中的 Beta 版以来，Pod 安全准入没有什么变化。重点是改善用户体验，同时继续保持高质量的标准。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:52]:
```suggestion
自 Kubernetes v1.23 中的 Beta 版以来，Pod 安全准入的变化不大。重点是改善用户体验，同时继续保持高质量的标准。
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+### Improved violation messages
+
+We improved violation messages so that you get
+[fewer duplicate messages](https://github.com/kubernetes/kubernetes/pull/107698). For example,
+instead of the following message when the Baseline and Restricted policies check the same
+capability:
+-->
+### 改进的违规消息
+
+我们改进了违规消息，以便您获得[更少的重复消息](https://github.com/kubernetes/kubernetes/pull/107698)。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:64]:
```suggestion
我们改进了违规消息，因此你收到的[重复消息更少](https://github.com/kubernetes/kubernetes/pull/107698)。 
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+
+We improved violation messages so that you get
+[fewer duplicate messages](https://github.com/kubernetes/kubernetes/pull/107698). For example,
+instead of the following message when the Baseline and Restricted policies check the same
+capability:
+-->
+### 改进的违规消息
+
+我们改进了违规消息，以便您获得[更少的重复消息](https://github.com/kubernetes/kubernetes/pull/107698)。 
+例如，当基线和受限策略检查相同的功能时，而不是以下消息：
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:65]:
```suggestion
例如，当基线和受限策略检查相同的权能时不会出现以下消息：
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+
+```
+pods "admin-pod" is forbidden: violates PodSecurity "restricted:latest": non-default capabilities (container "admin" must not include "SYS_ADMIN" in securityContext.capabilities.add), unrestricted capabilities (container "admin" must not include "SYS_ADMIN" in securityContext.capabilities.add)
+```
+
+<!--
+You get this message:
+-->
+你会收到这样的信息：
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:74]:
```suggestion
而是会看到这样的消息：
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+violations and surfaces a [warning](/blog/2020/09/03/warnings/) if any are out of compliance. These
+[warnings are now aggregated](https://github.com/kubernetes/kubernetes/pull/105889) for pods with
+identical violations, making large namespaces with many replicas much more manageable. For example:
+-->
+
+### 改进的命名空间警告
+
+当你修改命名空间上的 `enforce` Pod 安全标签时，Pod 安全准入控制器会检查所有现有 Pod 是否存在违规行为，
+如果有任何违规行为，则会显示 [警告](/blog/2020/09/03/warnings/). 这些 [警告现已汇总](https://github.com/kubernetes/kubernetes/pull/105889)
+针对具有相同违规行为的 Pod，使得具有许多副本的大型命名空间更易于管理。例如：
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:94]:
```suggestion
如果有任何违规行为，则会显示[警告](/blog/2020/09/03/warnings/)。
针对具有相同违规行为的 Pod，这些[警告现已聚合](https://github.com/kubernetes/kubernetes/pull/105889)，
使得具有许多副本的大型命名空间更易于管理。例如：
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+Warning: frontend-h23gf2: allowPrivilegeEscalation != false
+Warning: myjob-g342hj (and 6 other pods): host namespaces, allowPrivilegeEscalation != false Warning: backend-j23h42 (and 1 other pod): non-default capabilities, unrestricted capabilities
+```
+
+<!--
+Additionally, when you apply a non-privileged label to a namespace that has been
+[configured to be exempt](https://kubernetes.io/docs/concepts/security/pod-security-admission/#exemptions),
+you will now get a warning alerting you to this fact:
+-->
+此外，当你将一个非特权标签应用于已经[配置为豁免](https://kubernetes.io/docs/concepts/security/pod-security-admission/#exemptions)
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:106]:
```suggestion
此外，当你将一个非特权标签应用于已经[配置为豁免](/zh-cn/docs/concepts/security/pod-security-admission/#exemptions)
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 9]
```diff
@@ -0,0 +1,170 @@
...
+-->
+### Pod 安全标准的更改
+
+<!--
+The [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/),
+which Pod Security admission enforces, have been updated with support for the new Pod OS
+field. In v1.25 and later, if you use the Restricted policy, the following Linux-specific restrictions will no
+longer be required if you explicitly set the pod's `.spec.os.name` field to `windows`:
+-->
+Pod 安全准入执行的 [Pod 安全标准](https://kubernetes.io/docs/concepts/security/pod-security-standards/)已经更新，
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:124]:
```suggestion
Pod 安全准入执行的 [Pod 安全标准](/zh-cn/docs/concepts/security/pod-security-standards/)已经更新，
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 10]
```diff
@@ -0,0 +1,170 @@
...
+支持新的 Pod OS 字段。在 v1.25 及更高版本中，如果你使用限制策略，如果你明确将 Pod 的 `.spec.os.name` 字段设置为 `windows`，
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:125]:
```suggestion
支持新的 Pod OS 字段。在 v1.25 及更高版本中，
如果你使用限制策略且明确将 Pod 的 `.spec.os.name` 字段设置为 `windows`，
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 11]
```diff
@@ -0,0 +1,170 @@
...
+* Capabilities - 要求在容器的 `capabilities` 字段中删除 `ALL` 功能
+
+<!--
+In Kubernetes v1.23 and earlier, the kubelet didn't enforce the Pod OS field.
+If your cluster includes nodes running a v1.23 or older kubelet, you should explicitly
+[pin Restricted policies](https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces)
+to a version prior to v1.25.
+-->
+在 Kubernetes v1.23 及更早版本中，kubelet 没有强制执行 Pod OS 字段。
+如果你的集群包含运行 v1.23 或更早版本的 kubelet 的节点，你应该明确地 [pin Restricted policies](https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces)
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:144]:
```suggestion
如果你的集群包含运行 v1.23 或更早版本的 kubelet 的节点，
你应该明确地[锁定受限的策略](/zh-cn/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces)
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 12]
```diff
@@ -0,0 +1,170 @@
...
+
+<!--
+For instructions to migrate from PodSecurityPolicy to the Pod Security admission controller, and
+for help choosing a migration strategy, refer to the
+[migration guide](https://kubernetes.io/docs/tasks/configure-pod-container/migrate-from-psp/).
+We're also developing a tool called
+[pspmigrator](https://github.com/kubernetes-sigs/pspmigrator) to automate parts
+of the migration process.
+-->
+有关从 PodSecurityPolicy 迁移到 Pod Security 准入控制器的说明，以及选择迁移策略的帮助，请参阅[迁移指南](https://kubernetes.io/docs/tasks/configure-pod-container/migrate-from-psp/)。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:160]:
```suggestion
有关从 PodSecurityPolicy 迁移到 Pod Security 准入控制器的说明以及选择迁移策略的帮助，
请参阅[迁移指南](/zh-cn/docs/tasks/configure-pod-container/migrate-from-psp/)。
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 13]
```diff
@@ -0,0 +1,170 @@
...
+我们还在开发一个名为 [pspmigrator](https://github.com/kubernetes-sigs/pspmigrator) 的工具来自动化部分迁移过程。
+
+<!--
+We'll be talking about PSP migration in more detail at our upcoming KubeCon 2022 NA talk,
+[*Migrating from Pod Security Policy*](https://sched.co/182Jx). Use the
+[KubeCon NA schedule](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/schedule/)
+to learn more.
+-->
+我们将在即将到来的 KubeCon 2022 NA 演讲中更详细地讨论 PSP 迁移，[*Migrating from Pod Security Policy*](https://sched.co/182Jx) 使用
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:169]:
```suggestion
我们将在即将到来的 KubeCon 2022 NA 演讲中更详细地讨论 PSP 迁移，
[**Migrating from Pod Security Policy**](https://sched.co/182Jx) 使用
```

**Review by windsonsea — COMMENTED** [review]
1. 长句请适当折行
2. 如果 link 的页面已汉化，请相应修改路径为 /zh-cn/

**Comment by windsonsea** [comment]
LGTM

You can click `Resolve conversation` after modification so everyone will know the text have been changed according to the comments.

Preview FYI: https://deploy-preview-37616--kubernetes-io-main-staging.netlify.app/zh-cn/blog/2022/08/25/pod-security-admission-stable/

**Comment by sarazqy** [comment]
> LGTM
> 
> You can click `Resolve conversation` after modification so everyone will know the text have been changed according to the comments.
> 
> Preview FYI: https://deploy-preview-37616--kubernetes-io-main-staging.netlify.app/zh-cn/blog/2022/08/25/pod-security-admission-stable/

thank you, but still no lglm tag

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 14]
```diff
@@ -0,0 +1,178 @@
...
+<!--
+The Pod Security admission controller replaces PodSecurityPolicy, making it easier to enforce predefined
+[Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) by
+simply adding a label to a namespace. The Pod Security Standards are maintained by the K8s
+community, which means you automatically get updated security policies whenever new
+security-impacting Kubernetes features are introduced.
+-->
+Pod 安全准入控制器取代了 PodSecurityPolicy，只需向命名空间添加标签，
+就可以更轻松地执行预定义的 [Pod 安全标准](/zh-cn/docs/concepts/security/pod-security-standards/)。
+Pod 安全标准由 K8s 社区维护，这意味着每当引入新的影响安全的 Kubernetes 功能时，你都会自动获得更新的安全策略。
```
Comment (howieyuen) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:44]:
```suggestion
Pod 安全标准由 K8s 社区维护，这意味着每当引入新的影响安全的 Kubernetes 功能时，你都会自动获得更新后的安全策略。
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 15]
```diff
@@ -0,0 +1,178 @@
...
+**Authors:** Tim Allclair (Google), Sam Stoelinga (Google)
+-->
+**作者：** Tim Allclair (Google)，Sam Stoelinga (Google)
+
+<!--
+The release of Kubernetes v1.25 marks a major milestone for Kubernetes out-of-the-box pod security
+controls: Pod Security admission (PSA) graduated to stable, and Pod Security Policy (PSP) has been
+removed.
+-->
+Kubernetes v1.25 的发布标志着 Kubernetes 开箱即用的 Pod 安全控制的一个重要里程碑：Pod 安全准入（PSA）逐渐稳定，
```
Comment (howieyuen) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:25]:
```suggestion
Kubernetes v1.25 的发布标志着 Kubernetes 开箱即用的 Pod 安全控制的一个重要里程碑：Pod 安全准入（PSA）毕业至稳定，
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 16]
```diff
@@ -0,0 +1,178 @@
...
+我们还在开发一个名为 [pspmigrator](https://github.com/kubernetes-sigs/pspmigrator) 的工具来自动化部分迁移过程。
+
+<!--
+We'll be talking about PSP migration in more detail at our upcoming KubeCon 2022 NA talk,
+[*Migrating from Pod Security Policy*](https://sched.co/182Jx). Use the
+[KubeCon NA schedule](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/schedule/)
+to learn more.
+-->
+我们将在即将到来的 KubeCon 2022 NA 演讲中更详细地讨论 PSP 迁移，
+[**Migrating from Pod Security Policy**](https://sched.co/182Jx) 使用
```
Comment (howieyuen) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:177]:
```suggestion
[**从 Pod 安全策略迁移**](https://sched.co/182Jx)。使用
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 17]
```diff
@@ -0,0 +1,178 @@
...
+[KubeCon NA schedule](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/schedule/) 了解更多信息。
```
Comment (howieyuen) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:178]:
```suggestion
[KubeCon NA 日程表](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/schedule/) 了解更多信息。
```

**Review by howieyuen — COMMENTED** [review]
some minor tips, and this blog [Warning: Helpful Warnings Ahead](https://kubernetes.io/blog/2020/09/03/warnings/) doesn't has a zh-cn version

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 18]
```diff
@@ -0,0 +1,178 @@
...
+### Improved violation messages
+
+We improved violation messages so that you get
+[fewer duplicate messages](https://github.com/kubernetes/kubernetes/pull/107698). For example,
+instead of the following message when the Baseline and Restricted policies check the same
+capability:
+-->
+### 改进的违规消息
+
+我们改进了违规消息，因此你收到的[重复消息减少](https://github.com/kubernetes/kubernetes/pull/107698)。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:66]:
```suggestion
我们改进了违规消息，因此你收到的[重复消息会减少](https://github.com/kubernetes/kubernetes/pull/107698)。 
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 19]
```diff
@@ -0,0 +1,178 @@
...
+-->
+### Pod 安全标准的更改
+
+<!--
+The [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/),
+which Pod Security admission enforces, have been updated with support for the new Pod OS
+field. In v1.25 and later, if you use the Restricted policy, the following Linux-specific restrictions will no
+longer be required if you explicitly set the pod's `.spec.os.name` field to `windows`:
+-->
+Pod 安全准入执行的 [Pod 安全标准](/zh-cn/docs/concepts/security/pod-security-standards/)已经更新，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:127]:
```suggestion
Pod 安全准入执行的 [Pod 安全标准](/zh-cn/docs/concepts/security/pod-security-standards/)已经被更新，
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 20]
```diff
@@ -0,0 +1,178 @@
...
+支持新的 Pod OS 字段。在 v1.25 及更高版本中，如果你使用限制策略，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:128]:
```suggestion
支持新的 Pod OS 字段。在 v1.25 及更高版本中，如果你使用 Restricted 策略，
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 21]
```diff
@@ -0,0 +1,178 @@
...
+如果你使用限制策略且明确将 Pod 的 `.spec.os.name` 字段设置为 `windows`，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:129]:
```suggestion
且明确地将 Pod 的 `.spec.os.name` 字段设置为 `windows`，
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 22]
```diff
@@ -0,0 +1,178 @@
...
+则不再需要以下 Linux 特定的限制：
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:130]:
```suggestion
则不再需要以下特定于 Linux 的限制：
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 23]
```diff
@@ -0,0 +1,178 @@
...
+
+<!--
+* Seccomp - The `seccompProfile.type` field for Pod and container security contexts
+* Privilege escalation - The `allowPrivilegeEscalation` field on container security contexts
+* Capabilities - The requirement to drop `ALL` capabilities in the `capabilities` field on containers
+-->
+* Seccomp - Pod 和容器安全上下文的 `seccompProfile.type` 字段
+* Privilege escalation - 容器安全上下文中的 `allowPrivilegeEscalation` 字段
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:138]:
```suggestion
* 特权级提升 - 容器安全上下文中的 `allowPrivilegeEscalation` 字段
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 24]
```diff
@@ -0,0 +1,178 @@
...
+* Capabilities - 要求在容器的 `capabilities` 字段中删除 `ALL` 功能
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:139]:
```suggestion
* 权能字 - 要求在容器的 `capabilities` 字段中删除 `ALL` 功能
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 25]
```diff
@@ -0,0 +1,178 @@
...
+
+<!--
+In Kubernetes v1.23 and earlier, the kubelet didn't enforce the Pod OS field.
+If your cluster includes nodes running a v1.23 or older kubelet, you should explicitly
+[pin Restricted policies](https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces)
+to a version prior to v1.25.
+-->
+在 Kubernetes v1.23 及更早版本中，kubelet 没有强制执行 Pod OS 字段。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:147]:
```suggestion
在 Kubernetes v1.23 及更早版本中，kubelet 没有强制使用 Pod OS 字段。
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 26]
```diff
@@ -0,0 +1,178 @@
...
+如果你的集群包含运行 v1.23 或更早版本的 kubelet 的节点，
+你应该明确地[锁定受限的策略](/zh-cn/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces)
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:149]:
```suggestion
你应该明确地[锁定 Stricted 策略](/zh-cn/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces)
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md [hunk 27]
```diff
@@ -0,0 +1,178 @@
...
+For instructions to migrate from PodSecurityPolicy to the Pod Security admission controller, and
+for help choosing a migration strategy, refer to the
+[migration guide](https://kubernetes.io/docs/tasks/configure-pod-container/migrate-from-psp/).
+We're also developing a tool called
+[pspmigrator](https://github.com/kubernetes-sigs/pspmigrator) to automate parts
+of the migration process.
+-->
+有关从 PodSecurityPolicy 迁移到 Pod Security 准入控制器的说明，以及选择迁移策略的帮助，
+准入控制器的说明以及选择迁移策略的帮助，
+请参阅[迁移指南](/zh-cn/do cs/tasks/configure-pod-container/migrate-from-psp/)。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:167]:
```suggestion
请参阅[迁移指南](/zh-cn/docs/tasks/configure-pod-container/migrate-from-psp/)。
```

File: content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md (hunk 16 above)
Comment (tengqm) [reply content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:177]: 这个演讲的题目倒是可以不翻译，因为其版本就是英文的，没有中文版本

Comment (tengqm) [reply content/zh-cn/blog/_posts/2022-08-25-pod-security/index.md:177]: 这里的 use 翻译成“使用”有点儿生硬。可以考虑翻译为“你可以通过 ... 来了解细节。”

**Comment by mengjiao-liu** [comment]
Can you address these comments?

**Comment by Sea-n** [comment]
Hey @sarazqy, you didn't respond for a while, should we close this PR?

**Comment by tengqm** [comment]
/close
Author is not responding to the feedbacks for a long time.
Feel free to reopen and move this forward.

**Pull request closed without merging**